core/docz-core/src/commands/dev.ts

```diff
@@ -44,7 +44,7 @@ export const dev = async (args: Arguments<any>) => {
 
   try {
     await dataServer.start()
-    if (args.open || args.o) openBrowser(`http://${config.host}:${config.port}`)
+    if (args.open) openBrowser(`http://${config.host}:${config.port}`)
```

Comment:
I think it's probably safe to remove `openBrowser`, but I'm not sure if there's any cases where this is useful.

Reply:
I think we need to leave one solution:
1. Webpack open.
2. openBrowser.

I do not know why they added openBrowser, perhaps for more flexible settings. 
I am for the decision @daiyam https://github.com/pedronauck/docz/pull/887.

Reply:
@kirbydcool Can you update the PR please? See https://github.com/pedronauck/docz/pull/891#discussion_r291796868

Reply:
I think the updated code is fine as it is. It will open a browser only if you add the option `open: true` in the config.